tests/integration/test_fim/test_registry/test_registry_file_limit/test_registry_limit_values.py

```diff
@@ -105,24 +100,16 @@ def get_configuration(request):
 
 # Functions
 
-
 def extra_configuration_before_yield():
     """Generate registry entries to fill database"""
-    reg1_handle = RegOpenKeyEx(registry_parser[KEY], sub_key_1, 0, KEY_ALL_ACCESS | KEY_WOW64_64KEY)
-
-    for i in range(0, int(file_limit_list[-1]) + 10):
-        modify_registry_value(reg1_handle, f'value_{i}', REG_SZ, 'added')
+    reg1_handle = create_registry(registry_parser[WINDOWS_HKEY_LOCAL_MACHINE], MONITORED_KEY, KEY_WOW64_64KEY)
 
     RegCloseKey(reg1_handle)
 
 
 # Tests
-
-
-@pytest.mark.parametrize('tags_to_apply', [
-    {'file_limit_registry_conf'}
-])
-def test_file_limit_values(tags_to_apply, get_configuration, configure_environment, restart_syscheckd):
+@pytest.mark.skipif(sys.platform=='win32', reason="Blocked by issue wazuh/wazuh #11819")
```

Comment:
The test is only for Windows, so you only need to add de general skip.

```suggestion
@pytest.mark.skip(reason="Blocked by issue wazuh/wazuh #11819")
```